enhancements/windows-containers/health-management.md

```diff
@@ -77,7 +79,8 @@ To enable this, WMCO will have these changes in responsibility:
 * WMCO will create and maintain a ConfigMap which provides WICD with the specifications for each Windows service that
   must be created on a Windows instance.
 * WMCO will invoke the `bootstrap` command of WICD when initially adding a Windows instance to the cluster. This will
-  take the worker ignition file as an input, and will not involve the services ConfigMap.
+  take the worker ignition file as an input, and will only involve the services ConfigMap if containerd is the container
+  runtime in order to use its defined service configuration.
```

Comment:
Lets make sure we think through how the `bootstrap` command is going to recognize containerd from the services ConfigMap.
Some options:
* By name?
* Runtime is recognized by priority zero? Is this overloading the priority field?
* Should we have a separate runtime field in the service object within the services array?
* Do we add a separate `runtime` key which contains a service object on the same level as `services` and `files`? We could also add the endpoint within this. So something like:
```
{
  files: []
  services; []
  runtime: {
    service: {containerd service spec}
    endpont: {\\.\pipe\containerd-containerd}
  }
}
```

I am leaning towards the last option, as it is only possible for there to be one runtime. What do you think?

Reply:
After discussing, we decided that containerd runtime and its config will be represented as just another service in the services array (no new key), and this service will have the highest priority.

Reply:
How will WICD if containerd is the container runtime? Won't it need to look at the ConfigMap to figure that out? IOW, the `bootstrap` command will need to involve / look at the ConfigMap always.